h/views/activity.py

```diff
@@ -233,6 +236,13 @@ def delete_lozenge(self):
     def toggle_tag_facet(self):
         return _toggle_tag_facet(self.request)
 
+    def _check_access_permissions(self):
+        if not self.request.has_permission('read', self.group):
```

Comment:
This is just fine, but it's worth noting that strictly I think the `self.group` argument here isn't needed, as the default context is the request context, which here is the group already, right?